### DISCUSSION TITLE
New in Entity Framework 8: Primitive collection properties

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#315] [Announcements]
# New in Entity Framework 8: Primitive collection properties

New in Entity Framework 8: Primitive collection properties

https://timdeschryver.dev/bits/new-in-entity-framework-8-primitive-collection-properties

**Comment by sebdesalvador** [comment]
How do you configure the EF side?

**Reply by timdeschryver** [reply]
Hey @sebdesalvador , this configuration will automatically be applied in EF8.

**Reply by sebdesalvador** [reply]
How do you mean automatic? I’m a big fan of the fluent API, should I not use something like ˋbuilder.Property(x => x.MyCollection)`? Or ˋbuilder.PrimitiveCollection(x => x.MyCollection)`?

**Reply by timdeschryver** [reply]
Oh sorry, I misunderstood the question.
You still need to add the property to the entity mapping configuration, but this is the same as with other primitive properties.

From this example shared in the Bit, this means:

```
builder.Property(p => x.LuckyNumbers);
```
